src/BusinessLogic/CheckoutAPI/PromotionalWidgets/Responses/PromotionalWidgetsCheckoutResponse.php

```diff
@@ -35,13 +35,16 @@ public function toArray(): array
 
         return [
             'assetKey' => $this->widgetInitializer->getAssetKey(),
-            'merchantId' => $this->widgetInitializer->getMerchantId(),
+            'merchant' => $this->widgetInitializer->getMerchantId(),
```

Comment:
If I understand it well, this change breaks backward compatibility, so the new version will need to be 3.0.0.